Engines/Wine/Verbs/Remove Mono/script.js

```diff
@@ -0,0 +1,45 @@
+include(["engines", "wine", "engine", "object"]);
+include(["utils", "functions", "filesystem", "files"]);
+include(["engines", "wine", "plugins", "regedit"]);
+
+/**
+* Verb to remove mono
+* @returns {Wine} Wine object
+*/
+Wine.prototype.removemono = function () {
```

Comment:
I would recommend to name the method `removeMono` and its `id`: `remove_mono`

Reply:
an id cannot contain an _.

Reply:
Are you sure? What about:

https://github.com/PhoenicisOrg/scripts/blob/92fe6100542f3919e09cfd0de61575cee542cda9/Engines/Wine/Verbs/dotnet20/script.js#L5
 
and

https://github.com/PhoenicisOrg/scripts/blob/92fe6100542f3919e09cfd0de61575cee542cda9/Engines/Wine/Plugins/Windows%20version/script.json#L3

Reply:
OK, I remembered codacy complaining about a _ somewhere. So if it is not the id , it was certainly the method name ^^.

Reply:
We should have a documentation about this.